src/App.js

```diff
@@ -1,41 +1,63 @@
-import React,{useState} from "react";
+import React, { useState } from "react";
 import DiaryCard from "./Components/DiaryCard";
 import DiaryHome from "./Components/DiaryHome";
 
-
 function App() {
 
-
   const [title, setTitle] = useState("");
   const [description, setDescription] = useState("");
-  const [card,setCard] =useState(false);
- 
+
+  //State for show new card
+  const [card, setCard] = useState(false);
+
+  //Object Array to map cards
+  const [cards, setCards] = useState([])
+
   function onClick(e) {
     e.preventDefault();
 
-      if(title.trim().length === 0 || description.trim().length ===0){
-        console.log("Missing title or Missing Description");
+    if (title.trim().length === 0 || description.trim().length === 0) {
+      if (title.trim().length === 0) {
+        console.log("Title is missing")
       }
-      else{
-        setCard(true);
+      if (description.trim().length === 0) {
+        console.log("Description is missing")
       }
-   
+    }
+    else {
+      let newCardData = {"title" : title, "description": description}
+      let newArray =cards.concat(newCardData)
+      setCards(newArray)
```

Comment:
if you want to do modifications for current state values and again set modified values to same state use `setState(prev => prev)`  to access current state values. don't replace all values again. it is inefficient.